web/src/core/engines/Cesium/Feature/index.tsx

```diff
@@ -95,7 +95,7 @@ export default function Feature({
     k: keyof AppearanceTypes,
     f?: ComputedFeature,
   ): JSX.Element | null => {
-    const componentId = `${layer.id}_${f?.id ?? ""}_${k}_${isHidden}_${
+    const componentId = `${layer.id}_${f?.id ?? ""}_${k}_${isHidden}_${data?.url ?? ""}_${
```

Comment:
I think it should use `generateIDWithMD5` to compress string like [here](https://github.com/reearth/reearth/blob/02f87bcf8f63bbc45644643adf13d8bb500dbab7/web/src/core/engines/Cesium/Feature/index.tsx#L162).